whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/Inet6numSpec.groovy

```diff
@@ -2251,4 +2251,164 @@ class Inet6numSpec extends BaseQueryUpdateSpec {
+        then:
+        ack.summary.nrFound == 1
+        ack.summary.assertSuccess(0, 0, 0, 0, 0)
+        ack.summary.assertErrors(1, 1, 0, 0)
+
+        ack.countErrorWarnInfo(1, 0, 1)
+        ack.errors.any { it.operation == "Create" && it.key == "[inet6num] 2001:600::/25" }
+        ack.errorMessagesFor("Create", "[inet6num] 2001:600::/25") ==
+                ["Syntax error in not an url"]
+
+        queryObjectNotFound("-rGBT inet6num 2001:600::/25", "inet6num", "2001:600::/25")
+    }
+
+    def "create with not secure url as geofeed"() {
+        when:
+        def ack = syncUpdateWithResponse("""
+                inet6num:     2001:0600::/25
+                netname:      EU-ZZ-2001-0600
+                descr:        European Regional Registry
+                country:      EU
+                geofeed:      http://unsecure.com
+                org:          ORG-LIR1-TEST
+                admin-c:      TP1-TEST
+                tech-c:       TP1-TEST
+                mnt-by:       RIPE-NCC-HM-MNT
+                mnt-lower:    RIPE-NCC-HM-MNT
+                status:       ALLOCATED-BY-RIR
+                source:       TEST
+
+                password: hm
+                password: owner3
+                """.stripIndent())
+        then:
+        ack.summary.nrFound == 1
+        ack.summary.assertSuccess(0, 0, 0, 0, 0)
+        ack.summary.assertErrors(1, 1, 0, 0)
+
+        ack.countErrorWarnInfo(1, 0, 1)
+        ack.errors.any { it.operation == "Create" && it.key == "[inet6num] 2001:600::/25" }
+        ack.errorMessagesFor("Create", "[inet6num] 2001:600::/25") ==
+                ["Syntax error in http://unsecure.com"]
+
+        queryObjectNotFound("-rGBT inet6num 2001:600::/25", "inet6num", "2001:600::/25")
+    }
+
+    def "create with geofeed and inet6num too specific"() {
+        when:
+        def ack = syncUpdateWithResponse("""
+                inet6num:     2001:0600::/64
+                netname:      EU-ZZ-2001-0600
+                descr:        European Regional Registry
+                country:      EU
+                geofeed:      https://example.com
+                org:          ORG-LIR1-TEST
+                admin-c:      TP1-TEST
+                tech-c:       TP1-TEST
+                mnt-by:       RIPE-NCC-HM-MNT
+                mnt-lower:    RIPE-NCC-HM-MNT
+                status:       ALLOCATED-BY-RIR
+                source:       TEST
+
+                password: hm
+                password: owner3
+                """.stripIndent())
+        then:
+        ack.summary.nrFound == 1
+        ack.summary.assertSuccess(0, 0, 0, 0, 0)
+        ack.summary.assertErrors(1, 1, 0, 0)
+
+        ack.countErrorWarnInfo(1, 0, 1)
+        ack.errors.any { it.operation == "Create" && it.key == "[inet6num] 2001:600::/64" }
+        ack.errorMessagesFor("Create", "[inet6num] 2001:600::/64") ==
+                ["Adding the \"geofeed:\" attribute to an object with a prefix length greater than 48 is not allowed"]
```

Comment:
The test is for creating an inetnum but the message is for adding a geofeed attribute. I would make the validation message more generic so it applies to either a create or modify.

Reply:
Changed the messages, let me know if they look better.